### PR TITLE
Retro Cyberpunk

### PR DESCRIPTION
**Pull request by inescodes**
- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other files using the same name as yours?
- [x] Have you filled in at least the required data for `title`, `author`, and `stylesheet`?
- [x] Is the value of `stylesheet` a full URL that is publicly accessible and renders an unminified, compiled CSS file?
- [x] Do your styles pass proper contrast for accessibility?
- [x] Have you included `@media (prefers-reduced-motion: reduce)` to remove animations on `*,*::before,*::after`?
- [x] Did you style the `.skip-link`?
- [x] Have you checked on a real mobile device to ensure your responsive design doesn't cause overlaps or overflow scrolling?

### CODE REVIEW

File: src/_data/styles/retro-cyberpunk.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Retro Cyberpunk",
+  "author": "Inès K.",
+  "stylesheet": "https://codepen.io/inescodes/pen/ExKEpxG.css",
+  "twitter": "inescodes",
```
Comment (5t3ph) [inline src/_data/styles/retro-cyberpunk.json]:
```suggestion
  "twitter": "inescodes"
```

Just a little fixup so this is valid json 😊 don't worry, it's a common mistake!

**Review by 5t3ph — APPROVED** [review]
Excellent job @inescodes - really awesome work being mindful of contrast and motion 🙌

Thanks for participating! If you haven't already, subscribe to updates about new styles and features: https://stylestage.dev/subscribe/

**Comment by inescodes** [comment]
Thank you so much! Happy to participate :) Although I noticed that this version is a liiittle outdated. I made some small fixes but codepen didn't update the stylesheet, I'm not sure why.

**Comment by 5t3ph** [comment]
Hi @inescodes - the preview here will be a little older if that's what you looked at, but I just redeployed the live site so it could re-pull your styles so the live showcase will be your latest styles here shortly!

**Comment by inescodes** [comment]
I just checked it, the latest styles were there 👍 Thank you @5t3ph!

**Pull request merged**